xyz

```diff
@@ -119,8 +119,14 @@ next_version=$("$dir/node_modules/.bin/semver" -i "$increment" "$version") ||
 message="${message_template//X.Y.Z/$next_version}"
 tag="${tag_template//X.Y.Z/$next_version}"
 
-bold=$(tput bold)
-reset=$(tput sgr0)
+if [[ -t 1 ]] && [[ -n "$TERM" ]] && which tput &>/dev/null; then
```

Comment:
Okay. Shall we remove `[[ -n "$TERM" ]]` for the same reason?

Also, according to [Stack Overflow][1] we should avoid `which`. How about the following?

```bash
if type tput &>/dev/null ; then
  bold=$(tput bold)
  reset=$(tput sgr0)
fi
```

By the way, I did not know about `&>/dev/null`. Neat!


[1]: http://stackoverflow.com/a/677212/312785